packages/web3-eth-abi/types/index.d.ts

```diff
@@ -19,7 +19,7 @@
 
 import { AbiInput, AbiItem } from 'web3-utils';
 
-export class AbiCoder {
+declare class AbiCoder {
```

Comment:
Removing the export would be a breaking change, that's why the tests are also failing. Please fix it, add the entry in the CHANGELOG.md file as well.